lib/mini_profiler_rails/railtie.rb

```diff
@@ -24,7 +24,12 @@ def self.initialize!(app)
 
     c.skip_paths ||= []
 
-    c.skip_paths << app.config.assets.prefix if serves_static_assets?(app)
+    begin
+      c.skip_paths << app.config.assets.prefix if serves_static_assets?(app)
+    rescue
+      # app.config did not repond_to? :assets
```

Comment:
This feels odd its hiding a possible error + first line is !app.respond_to(:assets)